tests/delegated_scanning_test_utils.go

```diff
@@ -603,3 +629,64 @@ func (d *deleScanTestUtils) waitForBuildComplete(ctx context.Context, c buildv1c
 		}
 	}
 }
+
+// DisableMCONodeDrain configures the OpenShift Machine Config Operator to not drain
+// nodes when mirroring CRs are applied. This is best effort as the capability is
+// not available on every MCO z-stream release.
+//
+// This should speed up mirroring tests and reduce flakes on OCP versions that would
+// otherwise perform drains.
+//
+// The test will only fail if static testdata files are not found.
+//
+// TODO: Test on OCP 4.12, 4.13, 4.14, 4.15, 4.16, 4.17, 4.18
+func (d *deleScanTestUtils) DisableMCONodeDrain(t *testing.T, ctx context.Context) error {
+	// applyMachineConfiguration returns true if machine configuration CR was found.
+	applyMachineConfiguration := func() (bool, error) {
```

Comment:
What's the point of defining this as a func rather than just inlining the code? 🤔

Reply:
It originally was in-line, style wise I felt it reads better keeping errors aligned and 'handled' as they occur and avoided having to pre-define a few temp variables outside of the block.